public/index.html

```diff
@@ -28,7 +28,7 @@
   <meta name="twitter:description" content="" />
   <meta name="twitter:image" content="img/logo.png" />
 
-  <link rel="icon" href="img/logo.png" alt="Image Search App Logo" type="image/png" sizes="512X512">
+  <link rel="icon" href="img/logo.png" alt="Image Search App Logo" type="image/png" width="300" height="300">
```

Comment:
Someone is already fixing this in #35. Can you remove the changes for this part, please?